scripts/zones/Upper_Jeuno/npcs/Ajithaam.lua

```diff
@@ -45,7 +45,11 @@ Port Jeuno (West to East)
 ]]--
 
 entity.onTrade = function(player, npc, trade)
-    if (trade:getGil() == 300 and trade:getItemCount() == 1 and player:getQuestStatus(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.LURE_OF_THE_WILDCAT) == QUEST_COMPLETED and player:getCurrentMission(TOAU) > xi.mission.id.toau.IMMORTAL_SENTRIES) then
+    if trade:getGil() == 300 and
```

Comment:
Can you adjust this conditional to match style guide?
```
if
    <condition>
then
```

Reply:
oops, missed that.